packages/geo/src/lib/catalog/shared/catalog.service.ts

```diff
@@ -200,4 +202,36 @@ export class CatalogService {
     }
   }
 
+  private retriveLayerInfoFormat(catalogLayer: any, layersQueryFormat: {}[]) {
```

Comment:
Is this really of type any?

Reply:
Also, define a better type for layersQueryFormat. Not need for a dedicated interface but the inline typing could be improved.